keras_cv/metrics/coco/utils_test.py

```diff
@@ -21,19 +21,19 @@
 
 
 class UtilTest(tf.test.TestCase):
-    def test_filter_bounding_boxes_empty(self):
+    def DISABLE_test_filter_bounding_boxes_empty(self):
```

Comment:
Why do we have to disable those tests?

Reply:
COCO metrics are entirely disabled in this PR as there is:

- no way to make them work with the current implementation
- they still have numerical issues.

I figured it is better to keep this PR as small as possible and just disable COCO metrics, then re-enable them when they are fixed.